### DISCUSSION TITLE
Running chromVAR for a subset of peaks?

### DISCUSSION THREAD

**Discussion by rargelaguet** [GreenleafLab/ArchR#622] [Questions / Documentation]
Hello,
is it possible to run chromVAR for a subset of peaks? I could create submatrices of PeakMatrix and then run chromVAR for these, but it would  be more efficient if one could just specify a subset of peaks as argument to addDeviationsMatrix. Thanks!

**Comment by rcorces** [comment]
This is not currently possible as a parameter but could be done as you suggest using a manual work around. I imagine that if your peakset was X and you had a deviations matrix that was subsetX, this would probably break some downstream ArchR functionalities but I'm not 100% sure. Perhaps @jgranja24 can weigh in.
